app/konnect/runtime-manager/runtime-instances/index.md

```diff
@@ -2,3 +2,21 @@
 title: Runtime Instances Overview
 content_type: explanation
 ---
+
```

Comment:
What specifically about that information do you feel needs to be added to the runtime instances overview topic? 

Most of the information related to metrics is available in /konnect/analytics/summary-dashboard.

Reply:
@Guaris I think for me, I wouldn't necessarily look in /konnect/analytics/summary-dashboard or know to look there if I was wondering what the P99 latency was of a runtime instance. I guess my thought process was:

1. Looks at the Runtime Instance dashboard in Konnect.
2. What does P99 mean? 
3. Look in Runtime Manager section of docs for info about what that means in that context.

But I also don't know if P99 latency is sort of an industry standard type thing (along with the other metrics) that most people would know already before using Konnect.

Reply:
Hmm, I'm not sure if the Runtime Manager overview would be for the spot for it since you don't see any of those metrics on that page. But I see what you're saying about how it's not specific to just runtime instances. If the metrics here are the same as the ones in https://docs.konghq.com/konnect/analytics/summary-dashboard/, maybe we just add a link to that? Maybe what we need is a "Analytics Metrics" page or something that's a reference that explains the different analytics you can view in Konnect, but that's out of scope for this PR, I think.

Reply:
I added a link to the dashboard at the bottom.